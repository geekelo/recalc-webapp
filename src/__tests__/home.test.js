import React from 'react';
import { render } from '@testing-library/react';
import Home from '../componenets/home';

test('test calculator components work correctly', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});