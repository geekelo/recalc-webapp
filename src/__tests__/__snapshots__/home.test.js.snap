// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test calculator components work correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="overlay"
    >
      <div
        class="welcomeConstainer"
      >
        <h1>
          Welcome to Recalc
        </h1>
        <p>
          Recalc is your Ultimate Mathematical Companion! Are you tired of manually crunching numbers and wrestling with complex mathematical equations? Look no further! Recalc Calculator is here to revolutionize the way you perform calculations. With our user-friendly web app, you can now solve mathematical problems effortlessly and with unparalleled convenience.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
